docs/docs/bandits/config.mdx

```diff
@@ -0,0 +1,98 @@
+
+## Creating a Bandit
+
+Once Sticky Bucketing is enabled, you can now create a new Bandit from the Bandits page in the sidebar.
+
+### 1. Selecting an Assignment Attribute
+
+<img
+  src="/images/bandits/bandit-config-1.png"
+  alt="Configuring Bandit Hash Attributes and Number of Variations"
+  style={{ width: 600, margin: "0 auto" }}
+/>
+
+First, select the assignment Attribute used to randomize users into variations and that maps to the metrics you will use to track user behavior.
+
+:::note
+
+Bandits work best when your assignment Attribute (randomization ID) is consistent with the service you use for sticky bucketing. If you use a cookie-based sticky bucketing approach, we strongly recommend using a cookie-based identifier type for randomization (e.g. anonymous IDs), or else you may have Multiple Exposures problems in your Bandit.
+
+:::
+
+### 2. Setting the Number of Variations
+
+Select the number of variations you want to set up. The Bandit works well in cases where you have 4 or more arms and you're looking to weed out poor performers quickly, but if you have too many arms for your amount of traffic, then it may take a long time for the Bandit to identify a winner.
+
+### 3. Selecting a Decision Metric
+
+The Decision Metric is the metric that the Bandit will optimize towards. After each **reweighting** update, the Bandit will assign more traffic to the variations that are performing better on the Decision Metric. It should be a metric that you care about improving and that you can measure quickly after the user is exposed to the experiment.
+
+An ideal decision metric has the following properties:
+
+- It has a **short conversion window** and can be observed in a relatively short time frame. Short conversion windows allow the bandit to update with accurate information; a metric that takes longer to be observed will slow the Bandit down.
+- If it is a proportion metric, the conversion rate should not be super low or super high. These extreme conversion rates can result in Bandits that are very slow to converge as a lot of data is needed to adequately separate the Bandit variations.
+
+<img
+  src="/images/bandits/bandit-config-2.png"
+  alt="Configuring Bandit Decision Metric and Exploration Window"
+  style={{ width: 600, margin: "0 auto" }}
+/>
+
+### 4. Set the exploration window and update cadence
+
+The Bandit needs to know how long it should collect data for before beginning to update variation weights. This is called the **Exploration Window**. After the Exploration Window, the Bandit will begin to update variation weights at a set **Update Cadence**.
+
+**Exploration Window**
+
+Your Exploration Window should be long enough that you get enough traffic into each arm before the Bandit begins optimizing traffic. We _require_ at least 100 users per variation before we make any updates to our Bandit.
```

Comment:
"updates to our Bandit" could be more explicit, I think. Does "any updates" mean "before we reallocate traffic?"